presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -483,10 +483,10 @@ public boolean supportClientTransaction()
      * Extracts all values defined in the X-Presto-Client-Capabilities header.
      * <p>
      * The values are defined as a comma-separated list and they are used when it is necessary to specify
-     * which are the functions that client support.
+     * which are the supported functions for the client that request to the server.
```

Comment:
```suggestion
     * which are the supported functions for the user requester.
```

Reply:
I will put "presto client requester". Because the client is referring to a presto client not to a user